tests/sagemaker/test_deployment.py

```diff
@@ -165,13 +165,56 @@ def test_attempting_to_deploy_in_asynchronous_mode_without_archiving_throws_exce
     assert exc.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
 
 
+@mock_sagemaker_aws_services
+def test_deploy_creates_sagemaker_and_s3_resources_with_expected_names_from_local(
+        pretrained_model, sagemaker_client):
+    local_model_path = _download_artifact_from_uri(pretrained_model.model_uri)
+    app_name = "test-app"
+    mfs.deploy(app_name=app_name,
+               model_uri=local_model_path,
+               mode=mfs.DEPLOYMENT_MODE_CREATE)
+
+    assert any([app_name in model["ModelName"]
+                for model in sagemaker_client.list_models()["Models"]])
+    assert any([app_name in config["EndpointConfigName"]
+                for config in sagemaker_client.list_endpoint_configs()["EndpointConfigs"]])
+    assert app_name in [endpoint["EndpointName"]
+                        for endpoint in sagemaker_client.list_endpoints()["Endpoints"]]
+
+
+# @mock_sagemaker_aws_services
+# def test_deploy_creates_sagemaker_and_s3_resources_with_expected_names_from_s3(
+#         pretrained_model, sagemaker_client):
+#
+#     # TODO(sueann): this case is meant to test using a remote filestore URI as the model_uri
```

Comment:
if this is tricky we can discuss whether we really need this test - there are already tests for `RunsArtifactRepository` that uses `S3ArtifactRepository` internally, but it'd be good to make sure we don't do anything weird to the URI in the deployment module.